src/main/drivers/pwm_output.c

```diff
@@ -779,4 +773,14 @@ void beeperPwmInit(const ioTag_t tag, uint16_t frequency)
     }
 }
 #endif // USE_BEEPER
+#endif //USE_PWM_OUTPUT
+
+bool isMotorProtocolDshot(void)
```

Comment:
This should be conditional to USE_PWM_OUTPUT. If there is no PWM output, the concept of motor protocol is not defined, so places calling this in this case need to be fixed.

Reply:
done, see additional commit.